index.html

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>판다마켓</title>
+
+  <link rel="stylesheet" href="index.css"/>
+  <link rel="stylesheet" as="style" crossorigin href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.8/dist/web/static/pretendard.css" />
+
+</head>
+<body>
+  <header>
+    <a class="logo" href="/">
+      <img class="logo-img" src="img/logo.svg">
+      <img class="logo-title" src="img/logo-title.svg">
+    </a>
+    
+    <div class="login">
+      <a class="login-btn" href="/login.html">로그인</a>
+    </div>
+  </header>
+
+  <section class="outside top">
+    <div class="top-contents">
+      <div class="top-wording">
+        <h2 class="top-wording-title">
+          일상의 모든 물건을<br>
+          거래해 보세요
+        </h2>
+        <a class="top-wording-btn" href="/items.html">구경하러 가기</a>
+      </div>
+  
+      <img class="top-img" src="img/outside-top.svg">
+    </div>
+  </section>
+
+  <section>
+    <div class="info-container">
+      <img src="img/info1.svg">
+      
+      <div class="info-wording">
+        <h5 class="info-type">Hot item</h5>
+        <h2 class="info-title">
+          인기 상품을<br>
+          확인해 보세요
+        </h2>
+        <h3 class="info-sub">
+          가장 HOT한 중고거래 물품을<br>
+          판다 마켓에서 확인해 보세요
+        </h3>
+      </div>
+    </div>
+  </section>
+
+  <section>
+    <div class="info-container">
+      <div class="info-wording sec">
+        <h5 class="info-type">Search</h5>
+        <h2 class="info-title">
+          구매를 원하는<br>
+          상품을 검색하세요
+        </h2>
+        <h3 class="info-sub">
+          구매하고 싶은 물품은 검색해서<br>
+          쉽게 찾아보세요
+        </h3>
+      </div>
+
+      <img src="img/info2.svg">
+    </div>
+  </section>
+
+  <section>
+    <div class="info-container">
+      <img src="img/info3.svg">
+      
+      <div class="info-wording">
+        <h5 class="info-type">Register</h5>
+        <h2 class="info-title">
+          판매를 원하는<br>
+          상품을 등록하세요
+        </h2>
+        <h3 class="info-sub">
+          어떤 물건이든 판매하고 싶은<br>
+          상품을 쉽게 등록하세요
+        </h3>
+      </div>
+    </div>
+  </section>
+
+  <section class="outside bottom">
+    <div class="bottom-contents">
+      <div class="bottom-wording">
+        <h2 class="bottom-wording-title">
+          믿을 수 있는<br>
+          판다마켓 중고 거래
+        </h2>
+      </div>
+  
+      <img class="bottom-img" src="img/outside-bottom.svg">
+    </div>
+  </section>
+
+  <footer>
+    <div class="footer-container">
+      <p class="footer-left">©codeit - 2024</p>
+
+      <div class="footer-mid">
+        <a href="./privacy.html">Privacy Policy</a>
+        <a href="./faq.html">FAQ</a>
+      </div>
+
+      <div class="footer-right">
+        <a href="https://www.codeit.kr/" target="_blank"><img src="img/ic_facebook.svg"></a>
```

Comment:
```suggestion
        <a href="https://www.codeit.kr/" target="_blank"><img src="img/ic_facebook.svg" rel="noopener noreferrer"></a>
```
`target="_blank"` 를 사용하여 새 탭이나 새 창에서 링크를 열 때, 열린 새 페이지는 window.opener 객체를 통해 원래 페이지에 접근할 수 있습니다. 이는 잠재적인 보안 위험으로 아래처럼 rel="noopener"를 사용하여 window.opener 참조를 제거를 통해 원래 페이지에 접근할 수 없도록 설정합니다.